crates/sequencer/src/lib.rs

```diff
@@ -7,5 +7,54 @@ mod rpc;
 mod sequencer;
 mod utils;
 
+use std::net::SocketAddr;
+
 pub use config::{SequencerConfig, SequencerMempoolConfig};
 pub use sequencer::CitreaSequencer;
+use sov_modules_rollup_blueprint::RollupBlueprint;
+use sov_modules_stf_blueprint::StfBlueprint;
+use tokio::sync::oneshot;
+use tracing::{instrument, Instrument};
+
+/// Sequencer stf runner
+pub struct Sequencer<S: RollupBlueprint> {
```

Comment:
Out of curiosity:

Since we are now moving forward with `CitreaRollupBlueprint`, wouldn't it be better to use that extended trait instead of `RollupBlueprint`?

Reply:
We could. However, it wouldn't make so much of a difference unless we're overriding methods from the old trait.